lib/exchange.js

```diff
@@ -101,18 +101,24 @@ const exchange = (name, type, exchangeOptions) => {
         publish(msg, opts);
     };
 
-    const rpcServer = (key, handler) => {
+    const rpcServer = (key, handler,options) => {
 
         if (!replyQueue) {
             throw new Error('replyQueue not set - ensure { noReply: false } is passed to exchange options');
         }
 
         const rpcQueue = createQueue({
-            key,
+          ...{
+            key: key,
             name: key,
             prefetch: 1,
             durable: false,
-            autoDelete: true
+            autoDelete: false,
```

Comment:
why autodelete false?  if it false so why durable isn't true?

Reply:
durable is good when we want messages to stay in case of a rabbit crash . as we keep messages for a short time here  we probably wont want to process them once they are back